<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>919</width>
    <height>632</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-30</y>
      <width>921</width>
      <height>661</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/Images/Images/Menu/hs.jpg</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>151</width>
      <height>91</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Create a profile</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/Images/Images/Buttons/createprofile.png</normaloff>:/Images/Images/Buttons/createprofile.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>150</width>
      <height>150</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>420</y>
      <width>151</width>
      <height>91</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Select your level</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/Images/Images/Buttons/levelselection - Copy.png</normaloff>:/Images/Images/Buttons/levelselection - Copy.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>150</width>
      <height>150</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>520</y>
      <width>151</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/Images/Images/Buttons/highscore.png</normaloff>:/Images/Images/Buttons/highscore.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>150</width>
      <height>150</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_12">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>560</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Delete current profile</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/Images/Images/Buttons/delete.png</normaloff>:/Images/Images/Buttons/delete.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>320</y>
      <width>151</width>
      <height>91</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Load an existing save</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/Images/Images/Buttons/loadprofile.png</normaloff>
      <selectedon>:/Images/Images/Buttons/levelselectionblue - Copy.png</selectedon>:/Images/Images/Buttons/loadprofile.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>150</width>
      <height>150</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>20</y>
      <width>371</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/Images/Images/Menu/robotmazegold.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>370</y>
      <width>151</width>
      <height>251</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/Images/Images/Robot/nadia robot-01gold.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/Images/Images/Buttons/volume.png</normaloff>:/Images/Images/Buttons/volume.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>150</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>420</y>
      <width>171</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/Images/Images/Buttons/multiplayer.png</normaloff>:/Images/Images/Buttons/multiplayer.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>150</width>
      <height>150</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
